src/constants.cpp

```diff
@@ -2215,15 +2261,17 @@ const char *decap_cyber_types[] = {
   "balance tail",
   "body compartment",
   "cyber fin",
-  "cyber skull",
-  "cyber torso",
   "dermal sheathing",
   "foot anchor",
   "hydraulic jack",
   "move by wire",
   "climbing claws",
   "smartlink",
-  "muscle replacement"
+  "muscle replacement",
+  "cyberarms modifications",
+  "cyberlegs modifications",
+  "cyberskull modifications",
+  "cybertorso modifications"
```

Comment:
If cyberarm gyromount is a completely separate piece of 'ware that's not a mod, then it should be included here; otherwise it should be removed from the above cyber_types table.

Reply:
Completely seperate (and hopefully eventually functional) ware